app/models/post.rb

```diff
@@ -3,6 +3,11 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  # Validations
+  validates :Title, presence: true, length: { minimun: 3, maximum: 250 }
```

Comment:
[REQUIRED] There is a mistake in your code. You typed `minimun`  instead of `minimum`

Reply:
Thank you! I did not notice that, I will correct it.